src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -283,8 +283,6 @@ private void updatePrivilegeInSessionLevel(String sectionName, String sessionNam
     }
     
     /**
-     * used for bulk update of privileges for sectionName
```

Comment:
> For me disallowing empty @return is still the best alternative since "coding standard" is not a bible.

Did you mean you prefer this? That's my preference too, but I'm not sure if that's what you meant.

```java
/**
 * Gets the sum of two numbers passed as parameters.
 * 
 * @param n1 the first number
 * @param n2 the second number
 */
public void getSum(int n1, int n2) {
    return n1 + n2;
}
```

Regarding `Returns` vs `@returns`, my initial intuition was to prefer `@return` as it is more formal. However, the Google coding standard gives a legit reason for preferring `Returns` which made me change my preference as well. The reason given is that in some contexts, overview statement is the only documentation shown and therefore it takes priority over `@return`.